app/src/main/java/com/example/plantsapp/data/firebase/repository/FirebasePlantsRepository.kt

```diff
@@ -49,13 +67,20 @@ class FirebasePlantsRepository @Inject constructor(
             .get()
             .await()
             .toObjects<FirebasePlant>()
-            .map { it.toPlant() }
+            .map {
+                it.toPlant(
```

Comment:
Is in necessary to save image to local storage? 
Maybe you can just use url?

https://firebase.google.com/docs/storage/android/upload-files#kotlin+ktx

<img width="705" alt="image" src="https://user-images.githubusercontent.com/23121846/144822726-561a567c-0f48-4cbd-8731-6647d5b1097b.png">